src/features/todos.ts

```diff
@@ -4,5 +4,10 @@ import { Todo } from '../types/Todo';
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    add: (todos, payload) => [...todos, ...payload.payload],
```

Comment:
The parameter `payload` in the reducer function should be named `action` to align with Redux Toolkit's convention, where the second parameter is the action object. This will make the code more readable and consistent with common practices.